### ISSUE TITLE
Over-large bounding box check breaks rendering on 4K screens

### ISSUE THREAD

**Issue by wm4** [libass/libass#220]
This funny check:

``` C
    if (w < 0 || h < 0 || w > 8000000 / FFMAX(h, 1) ||
        w > INT_MAX - 2 * bord || h > INT_MAX - 2 * bord) {
```

can break rendering on very large screens, as evidenced by mpv-player/mpv#3070.

**Comment by grigorig** [comment]
Not sure how to handle this. I guess such a check can be useful in case of overflow errors and the like, or bugs in some of the layout code. Maybe just increase the limit?

**Comment by wm4** [comment]
Does anyone know why 8000000 was chosen?

**Comment by grigorig** [comment]
I think this is old heritage back from the MPlayer days.

**Comment by rcombs** [comment]
Oh, #260 is a dupe of this; it's fixed in aa7aa03609fa6ab6f3a51b6f6a8d9e2dfa509ad2.
